go.mod

```diff
@@ -24,11 +25,11 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.0.1
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/fsnotify.v1 v1.4.7
-	k8s.io/api v0.17.7
-	k8s.io/apiextensions-apiserver v0.17.7
-	k8s.io/apimachinery v0.17.7
-	k8s.io/client-go v0.17.7
-	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
+	k8s.io/api v0.17.8
+	k8s.io/apiextensions-apiserver v0.17.8
+	k8s.io/apimachinery v0.17.8
+	k8s.io/client-go v0.17.8
+	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19
```

Comment:
This got downgraded?

Reply:
It's newer `2020 06 03` and `2020 06 19`  in this addition